src/EditorFeatures/CSharpTest/SpellCheck/SpellCheckSpanTests.cs

```diff
@@ -101,125 +101,173 @@ class {|Identifier:C|}
         [Fact]
         public async Task TestString1()
         {
-            await TestAsync(@"{|String:"" goo ""|}");
+            await TestAsync(@""" {|String:goo|} """);
         }
 
         [Fact]
         public async Task TestString2()
         {
-            await TestAsync(@"{|String:"" goo |}");
+            await TestAsync(@""" goo ");
```

Comment:
we no longer report anything on string literals taht have compiler syntax errors on them.  that's because we don't know what their content means, so we can't effectively break them into pieces to spell check.